apps/asap-cli/src/scripts/algolia/tests/process-productivity-metrics.test.ts

```diff
@@ -0,0 +1,252 @@
+import { SearchResponse } from '@algolia/client-search';
+import { teamOutputDocumentTypes, timeRanges } from '@asap-hub/model';
+import type { SearchIndex } from 'algoliasearch';
+
+import {
+  deletePreviousObjects,
+  getAllHits,
+  getBellCurveMetrics,
+  getStandardDeviation,
+  processTeamProductivityMetrics,
+  processUserProductivityMetrics,
+} from '../process-productivity-metrics';
+
+describe('getStandardDeviation', () => {
+  it('should return 0 for an empty array', () => {
+    expect(getStandardDeviation([], 0)).toBe(0);
+  });
+
+  it('should return correct standard deviation for an array of numbers', () => {
+    const numbers = [1, 2, 3, 4, 5];
+    const mean =
+      numbers.reduce((sum, value) => sum + value, 0) / numbers.length;
+    expect(getStandardDeviation(numbers, mean)).toBeCloseTo(1.414, 3);
+  });
+});
+
+describe('getBellCurveMetrics', () => {
+  it('should return correct bell curve metrics', () => {
+    const data = [1, 2, 3, 4, 5];
```

Comment:
would be nice to have a data set where below and above average max and min define a range, not just one number